static/js/components/AssignmentList.jsx

```diff
@@ -15,20 +15,21 @@ const AssignmentList = ({ assignments }) => {
   const { users } = useSelector((state) => state);
   const { observingRunList } = useSelector((state) => state.observingRuns);
   const { instrumentList } = useSelector((state) => state.instruments);
+  const [loadingUsers, setLoadingUsers] = useState([]);
```

Comment:
```suggestion
  const [loadingOrLoadedUsers, setLoadingOrLoadedUsers] = useState([]);
```
So that we know it's not just for in-progress queries